.github/workflows/build-publish-docker-image.yml

```diff
@@ -0,0 +1,65 @@
+name: Build and push Docker image to IBM container registry
+
+on:
+  workflow_call:
+    inputs:
+      DOCKER_IMAGE_NAME:
+        required: true
+        type: string
+      IBMCLOUD_REGION:
+        required: true
+        type: string
+      IBMCLOUD_RESOURCE_GROUP:
+        required: true
+        type: string
+      PR_TAG:
+        required: true
+        type: string
+    secrets:
+      IBMCLOUD_ACCOUNT:
+        required: true
+      IBMCLOUD_API_KEY:
+        required: true
+
+jobs:
+  build-publish-docker-image:
+    runs-on: ubuntu-latest
+    env:
+      DOCKER_IMAGE_TAG: icr.io/community-digital/${{ inputs.DOCKER_IMAGE_NAME }}:${{ inputs.PR_TAG }}
```

Comment:
We should avoid hard-code `community-digital`, right?

Reply:
good catch, I agree